docs/snippets/common/badge-story.mdx.mdx

```diff
@@ -2,12 +2,15 @@
 
 <!---Badge.stories.mdx --->
 
+<!--- This is your Story template function, shown here in React -->
```

Comment:
Can we put this above the `export const Template = ` line?

Reply:
Updated with the latest commit